modular_scarletreach/code/modules/jobs/job_types/roguetown/adventurer/stigmata.dm

```diff
@@ -0,0 +1,141 @@
+/datum/advclass/cleric/stigmata
+	name = "Stigmata"
+	tutorial = "PSYDON weeps. You are a devout cleric of the Allfather whom takes the suffering of others upon themselves. You have eschewn violence. You will suffer. You will endure."
+	vampcompat = FALSE
+	outfit = /datum/outfit/job/roguetown/adventurer/stigmata
+	class_select_category = CLASS_CAT_CLERIC
+	category_tags = list(CTAG_ADVENTURER, CTAG_COURTAGENT)
+
+	subclass_stats = list(
+		STATKEY_CON = 5,
+		STATKEY_END = 3,
+		STATKEY_SPD = 1,
+		STATKEY_STR = -2,
+		STATKEY_INT = -2,
```

Comment:
why negative int on a casting class...int effects spell cooldowns

Reply:
ALSO a dodging class. Already just begging to be feinted

Reply:
int malus was an unintentional holdover from this being based off flagellant, have removed it